src/ToggleButton.js

```diff
@@ -71,15 +59,24 @@ class ToggleButton extends React.Component {
       value,
       disabled,
       inputRef,
-      innerRef,
       ...props
-    } = this.props;
-    const { focused } = this.state;
+    },
+    ref,
+  ) => {
+    const [focused, setFocus] = useState(false);
+
+    const handleFocus = e => {
```

Comment:
add `useCallback`

Reply:
I combined both focus/blur to a single toggle handler, since really it is just switching it between true/false. Let me know if this should be reverted to be more explicit (it seems like we should only be `onFocus` when it is currently blurred (`focused: false`) and `onBlur` when it is currently focused `focused: true`.